src/app/components/Staking/Form/States/Message.tsx

```diff
@@ -1,27 +1,30 @@
+import { Heading, Text } from "@babylonlabs-io/bbn-core-ui";
 import Image from "next/image";
 
 interface MessageProps {
   title: string;
-  messages: string[];
```

Comment:
Update this to just a string but we can still use the string literal template to do line breaks if we want to.